CONTRIBUTING.md

```diff
@@ -166,3 +166,9 @@ We use several labels to help organize and identify issues. You can find all lab
 * Make sure you have [Jekyll](https://jekyllrb.com/) installed (using `$ gem install jekyll bundler`).
 * Build and run the website locally using `$ jekyll serve`.
 * Connect to the website in your browser via the "Server address" provided by the output of this command, e.g. `http://localhost:4000/`
+
+## Building In Your Browser
+
+Alternatively, you can build and run the website in a readily configured online workspace:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io#https://github.com/simple-icons/simple-icons)
```

Comment:
I originally started with an in-text link, since I'd noticed that you don't embed images anywhere else in this file, however I felt that it made it less clear what the link's purpose was as opposed to a button.

I'm happy to revert to an in-text link if you and other maintainers prefer that, to be more consistent with the rest of the file and keep the server build instructions low-profile.